src/components/Header/groups/SwapNavGroup.tsx

```diff
@@ -124,7 +125,7 @@ const SwapNavGroup = () => {
             </StyledNavLink>
           )}
 
-          {isSolana || (
+          {(isSolana || chainId !== ChainId.LINEA_TESTNET) && (
```

Comment:
wrong condition, should be
!isSolana && chainId !== ChainId.LINEA_TESTNET

Reply:
nice catch, I've updated